src/System.Runtime.Serialization.Json/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uapaot;
```

Comment:
Do you mind running `msbuild /t:UpdateVSConfigurations`? We try to run that everytime that we do a `Configurations.props` change so that we don't break VS